lesson10/public/app.js

```diff
@@ -0,0 +1,51 @@
+$('#submit').click(()=>{
+  const name = $('#name').val();
+  const comment = $('#comment').val();
+  $.ajax({
+    type: 'POST',
+    url: '/nameCom',
+    data:JSON.stringify ({name, comment}),
+    success:(data)=>{
+      console.log('success:', data);
+    },
+    contentType: 'application/json'
+   })
+})
+
+/*chat function*/
+const chat =()=>{
+  $.get('/ncom.txt', (data)=>{
```

Comment:
ncom can be confusing. Especially in this case you know it's gonna be a chat history not just name and comments. Why not name it chatHistory